t/Makefile

```diff
@@ -45,9 +46,9 @@ test-commands : $(TEST_CMDS)
 
 test : test-commands
 	$(RM) -r remote test_count{,.lock}
-	@bash -c ". ./testenv.sh && setup && cd t && \
+	@. ./testenv.sh && setup && cd t && \
 		RM_GIT_LFS_TEST_DIR=no $(PROVE) $(PROVE_EXTRA_ARGS) t-*.sh && \
-		shutdown"
+		shutdown
```

Comment:
Thanks so much for adding this change!  It's great to be able to take advantage of the fixed shell environment here.